### DISCUSSION TITLE
iOS: Add Reply option to "..." Menu that starts with "@username"

### DISCUSSION THREAD

**Discussion by chrisballinger** [community/community#13466] [Mobile]
When replying to PR and issue comments in the iOS app, the only option in the "..." menu is "Quote Reply" which does not include the username of the person you're replying to in the text. It would be much better if:

1) Quote Reply included the @username of the person you're replying to so they get a notification
2) A regular Reply option starting with @username when a quoted reply is not necessary

The workflow right now means I need to manually type out that person's username.

Thanks!

**Comment by aconsuegra** [comment]
Hey @chrisballinger, thanks for the feature request! I agree that this would be a nice improvement to the reply/quote flow in the apps, so I'm creating an internal issue to track it. Keep the feedback coming!
